deployments/dev/environment.py

```diff
@@ -33,6 +33,8 @@ def env() -> Mapping[str, Optional[str]]:
         'AZUL_DSS_DIRECT_ACCESS_ROLE': 'arn:aws:iam::109067257620:role/azul-sc',
         'AZUL_SUBSCRIBE_TO_DSS': '0',
 
+        'AZUL_TDR_BIGQUERY_DATASET': 'gbq:broad-jade-dev-data:dataset/datarepo_hca_mvp',
```

Comment:
```suggestion
        'AZUL_TDR_DATASET': 'gbq:broad-jade-dev-data:dataset/datarepo_hca_mvp',
```

Reply:
I don't believe this is accurate given your previous comment. Sticking to TDR naming conventions, this variable could be used to specify a snapshot, not a dataset. A TDR snapshot is a BigQuery dataset, but it is not a TDR dataset.

Reply:
I will remove the reference to BigQuery, but I think "TARGET" is the most accurate one-word solution (unless you'd prefer "DATASET_OR_SNAPSHOT").